### PR TITLE
Feature/add experiment metrics handlers

### PR DESCRIPTION
**Pull request by slawrence87544**
This pull request covers all changes required for GET/PUT request for the metric types table, the experiment metrics table, and the harvest controller.

### CODE REVIEW

File: src/expt_metrics.py [hunk 1]
```diff
@@ -0,0 +1,847 @@
+"""
+Copyright 2022 NOAA
+All rights reserved.
+
+Collection of methods to facilitate handling of score db requests
```
Comment (frolovsa) [inline src/expt_metrics.py]: maybe more specific on what type of requests?

File: src/harvest_controller.py [hunk 2]
```diff
@@ -0,0 +1,232 @@
+"""
+Copyright 2022 NOAA
+All rights reserved.
+
+Collection of methods to facilitate handling of score db requests
```
Comment (frolovsa) [inline src/harvest_controller.py]: please be more specific

File: tests/test_metric_types_handler.py [hunk 3]
```diff
@@ -0,0 +1,216 @@
...
+            'innov_stats_temperature_bias',
+            'temperature',
+            'celsius',
+            'bias',
+            description_temperature_bias
+        ),
+        MetricType(
+            'innov_stats_temperature_rmsd',
+            'temperature',
+            'celsius',
```
Comment (frolovsa) [inline tests/test_metric_types_handler.py:106]: change all Celsius to K

File: src/harvest_controller.py (hunk 2 above)
Comment (frolovsa) [reply src/harvest_controller.py]: also rename the file to be specific to innovation stats

**Review by frolovsa — CHANGES_REQUESTED** [review]
see comments. thank you for all this hard work.

File: src/score_table_models.py [hunk 4]
```diff
@@ -129,22 +129,39 @@ class Region(Base):
...
-    stat_type = Column(String(32))
-
-    # metrics = relationship('ExperimentMetric', backref='metric_type')
+    measurement_type = Column(String(64), nullable=False)
+    measurement_units = Column(String(64))
+    stat_type = Column(String(64))
+    description = Column(JSONB(astext_type=sa.Text()), nullable=True)
+    created_at = Column(DateTime, nullable=False)
+    updated_at = Column(DateTime)
+
```
Comment (frolovsa) [inline src/score_table_models.py:155]: Also need to add another column that will document the averaging period. E.g. for atm. it is 6 hours. For ocean it is 24 hours.

File: src/harvest_controller.py (hunk 2 above)
Comment (slawrence87544) [reply src/harvest_controller.py]: Done...

File: src/expt_metrics.py (hunk 1 above)
Comment (slawrence87544) [reply src/expt_metrics.py]: done...

File: src/score_table_models.py (hunk 4 above)
Comment (slawrence87544) [reply src/score_table_models.py:155]: We discussed this and came to the determination to assess this after using the tool for a trial period.

**Pull request merged**